Fix table of contents (installation section) in README.md

antonlogvinenko opened the pull request:
Links in TOC section describing installation are broken. I fixed the links, but I had to rename inner "Installation" section to "Installation with Shake" since there already exists "Installation" section (parent) and #installation link would wrongly refer to it.

Anrock commented:
Oops, my bad, totally forgot about TOC.

Wouldn't it be better to rename installation with Shake to `Building` or `Compiling` maybe? I specifically removed `with Shake` pieces everywhere as unneeded clarification after Makefile and powershell scripts mentions were removed.

Also, personally these installation sections inside installation section look confusing to me. Maybe they should be moved from under installation to top level? @alanz @fendor

antonlogvinenko commented:
> Wouldn't it be better to rename `installation with Shake` to `Building` or `Compiling` maybe? I specifically removed `with Shake` pieces everywhere as unneeded clarification after Makefile and powershell scripts mentions were removed.
Sure, I've updated PR and renamed to "Building".

> Also, personally these installation sections inside installation section look confusing to me. Maybe they should be moved from under installation to top level? @alanz @fendor
I thought about it, and noticed that the contents of "Building" section logically belong to "Installation from source" section. Maybe we should merge them? Then there will be just "Installation" section with three subsections "From Nix", "on ArchLinux", "From source" — these three then could be moved to top level.

Anrock commented:
@antonlogvinenko thanks for update, LGTM then.

Regarding sections: building section is definitely belongs to installation from source as a direct continuation after installing required libs and downloading source code.

antonlogvinenko commented:
> Regarding sections: building section is definitely belongs to installation from source as a direct continuation after installing required libs and downloading source code.
Ok, I moved "Building" section inside "Installation from source".

Anrock commented:
@antonlogvinenko thank you
